CDP4WebServices.API/Services/Operations/OperationProcessor.cs

```diff
@@ -392,6 +397,12 @@ internal void ValidateUpdateOperations(CdpPostOperation operation)
             {
                 throw new InvalidOperationException("FileRevisions updates are not allowed");
             }
+
+            //verify that a participant has a valid domain
+            if (operation.Update.Any(x => x[ClasskindKey].ToString() == typeof(Participant).Name) && operation.Update.Any(x => x[SelectedDomain].ToString() == Guid.Empty.ToString()))
```

Comment:
Sam proposed to create a side effect, so this code should be moved / refactored to somewhere els, but you could think of this for future development:

I think `typeof(Participant).Name` should be `nameof(Participant)`.

Also, `x[SelectedDomain]` could be `x[nameof(Participant.SelectedDomain)]`.
Then you don't need the `const` at all.